src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public int compareTo(Name other) {
```

Comment:
Useful sort function! Perhaps can further expand on the feature to sort not just by name, but maybe by address as well.